### PR TITLE
Fix blink transaction construction

### LINKED ISSUE #33: Blink Transactions not working on Android Wallet
Note 9 android + multiple other android devices issues present 
- Send Blink transactions fail, only able to make slow transactions. 
- Blink is best for Mobile use, please fix. 
- 
Blink mobile issue

### PR DESCRIPTION
**Pull request by jagerman**
The blink priority value was still set to the old magic value, and so the required burn amount wasn't getting added to the transaction data (and thus was getting rejected by blink quorums).

This commit fixes it; it also slightly changes the logic to match loki-core: we have "slow" priority, and anything else is blink.  (In practice this shouldn't matter since only the two priorities--slow and blink--weere exposed, but just makes the logic consistent).

(It also changes a few random project settings that were needed to build it in the latest version of android studio).

Fixes #33

### CODE REVIEW

**Comment by jagerman** [comment]
Fixed #33

**Comment by jagerman** [comment]
Fixes #33

File: gradle.properties [hunk 1]
```diff
@@ -15,6 +15,3 @@ org.gradle.jvmargs=-Xmx2048m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
-
-# keep build timestamps in APK
-android.keepTimestampsInApk = true
```
Comment (Mikunj) [inline gradle.properties:20]: Hmm why do we need to remove this?

Comment (jagerman) [reply gradle.properties:20]: The latest android studio version (4.0, I think?) refused to build with it in place, saying that it had previously been deprecated and was now removed and refused to build with it present.

Comment (Mikunj) [reply gradle.properties:20]: Ah alright

**Review by Mikunj — APPROVED** [review]

**Pull request merged**